tools/Google.Cloud.Tools.SbomGenerator/Program.cs

```diff
@@ -132,7 +132,7 @@ private static Spdx SpdxFrom(string nupkgPath)
     {
         // Compute the SHA-256 hash of the .nupkg file.
         using var sha256 = SHA256.Create();
-        byte[]? hash = null;
+        byte[] hash = null;
```

Comment:
We don't need to initialize this at all... we don't read from it until after the `using` statement which it's assigned in.